docs/semgrep-cloud-platform/user-management.md

```diff
@@ -157,5 +157,24 @@ To **change a user's role**:
 You cannot change your own role.
 :::
 
+## Appendix: Token scopes
+
+When creating a new token, you are able to set a token's **scope**. Token scopes enable you to limit or grant permissions as necessary. Tokens are also autogenerated with appropriate scopes by Semgrep Cloud Platform based on the user's role or when onboarding a repository.
+
+The following table displays token scopes and their permissions:
+
+| Token scope | Send findings from a remote repository | Send findings from a local repository |  Send PR or MR comments | Connect to Semgrep API |
```

Comment:
Let me know if we have additional permissions! 

I might also switch the axes (switch headers on top row (X) with scopes on leftmost column (Y)).

Reply:
These look good to both me and @tpetr.